documentation/Geopotential.tex

```diff
@@ -1,8 +1,5 @@
 \documentclass[10pt, a4paper, twoside]{basestyle}
 
-\usepackage[backend=biber,firstinits=true,maxnames=100,style=alphabetic,maxalphanames=4,doi=true,isbn=false,url=false,eprint=true]{biblatex}
-\bibliography{bibliography}
-
```

Comment:
This changes the spacing between the bibliographical id and the bibliographical entry in the References section (previously the entry started roughly below the "n" of "References", now it's below the "e".  This less visually pleasant.  In particular in this document this leads to a [ridiculously overfull hbox](https://idf.social/system/media_attachments/files/000/003/016/original/a8cacb25eed2eeb3.png?1536755438) when in the `[Wes17]` reference.

Reply:
This was because of the `nocite{*}` in another `refsegment` (whose bibliography is not printed) in the same `refsection` (the margin shifted to accomodate the widest possible label).

This was a way to ensure globally consistent labels with automated disambiguation; now that we have `userd` we no longer need that.
Note that this means that global uniqueness of the references will be checked only when compiling `bibliography` (but we should probably make the CI do that anyway).

Done.

Reply:
There are still differences in the way that the URLs get folded in `[PL10]` and `[Wes17]` (and the latter is still ugly).

Reply:
Ah, this is because `eprint` now starts a line per the global style, and our current settings forbid mid-word linebreaking in a URL. I will try to tame the `hyperref` options.

Reply:
Done, there’s a `biblatex` setting for that (found in https://tex.stackexchange.com/a/134281).